src/plugins/api-proxy/util/receive.js

```diff
@@ -196,31 +200,32 @@ export const makeLogResponse = request => ([response, body]) => {
 		statusCode >= 500 || // REST API had an internal error
 		(method.toLowerCase() === 'get' && statusCode >= 400) // something fishy with a GET
 	) {
-		if (statusCode === 400) {
-			console.info(`swallowing 400 error for ${pathname}`);
-			return;
+		// use console.error or console.warn to highlight these cases in Stackdriver
+		const errorMessageString = JSON.stringify({
+			message: 'REST API error response',
+			info: {
+				url: href,
+				query: parsedQuery,
+				method,
+				id,
+				statusCode,
+				time: elapsedTime,
+				originRequestId: request.id,
+				body,
+			},
+		});
+
+		// Only want to warn about 400s for now (maybe forever?)
+		if (statusCode < 500) {
```

Comment:
Truthfully, I'm not sure if new relic picks these up, but I think better to err on the side of caution.

Reply:
yeah i'm guessing it watches stderr